app/services/passporting/means_passporter.rb

```diff
@@ -27,7 +27,7 @@ def not_means_tested?
     end
 
     def age_passported?
-      parent_was_age_passported? || applicant_under18?
+      parent_was_age_passported? || age_passported_at_datestamp_or_now?
     end
 
```

Comment:
👇  I believe `parent_was_age_passported?` is not longer required however this PR is already big enough, so will ticket up looking into that another time.